assets/css/app.css

```
@@ -1,5 +1,326 @@
-@import "tailwindcss/base";
-@import "tailwindcss/components";
-@import "tailwindcss/utilities";
+@config "../tailwind.config.js";
+@import "tailwindcss";
```

Stylelint (17.12.0)

[error] Invalid position for `@import` rule (no-invalid-position-at-import-rule)

(no-invalid-position-at-import-rule)